File: lotto.py
import random
icnt = int(input("구매수량을 입력하시오 : "))
y = 1
while y <= icnt:
	lotto = []
	i = 1
	while i <= 45:
		lotto.append(i)
		i += 1
	cnt = 1
	lottonum =""
	while cnt <= 6:
		listSize = len(lotto) -1
		idx = random.randint(0, listSize)
		num = lotto.pop(idx)
		lottonum += str(num) +", "
		cnt += 1
	print(lottonum.rstrip(", "))
	y += 1;	

File: 20211108/src/Lotto.java

public class Lotto {
	public static void main(String[] args) {
		int lotto[] = new int[6];
		// 번호 생성
		for (int i = 0; i < 6; i++) {
			lotto[i] = (int) (Math.random() * 45) + 1;
			// 중복 번호 제거
			for (int j = 0; j < i; j++) {
				if (lotto[i] == lotto[j]) {
					i--;
					break;
				}
			}
		}
		System.out.print("로또 번호: ");

		// 번호 출력
		for (int i = 0; i < 6; i++) {
			System.out.print(lotto[i] + " ");
		}
	}
}